scripts/protocgen.sh

```diff
@@ -7,7 +7,7 @@ set -eo pipefail
 
 cd packages/proto
 # To also initialize, fetch and checkout any nested submodules
-git submodule update --init --recursive
+# git submodule update --init --recursive
```

Comment:
Why commented?

Reply:
Good catch, was for saving time of local tests. Fixed now